_posts/0000-01-02-Melowil.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!:
+---
+Your text
```

Comment:
This assignment looks hard do you want me to have my dog eat it ?

Reply:
I'm having trouble editing this file. I'm trying to replace line 5 with a meme and it keeps going to my comment section. 

Please help ! How do I edit this portion of the assignment ?